.github/workflows/changelog.yml

```
@@ -52,7 +52,7 @@ jobs:
           body: |-
             Thank you for your contribution! :rocket:
 
-            Please note that the `CHANGELOG.md` file contents are handled by the maintainers during merge. This is to prevent pull request merge conflicts, especially for contributions which may not be merged immediately. Please see the [Contributing Guide](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/CONTRIBUTING.md) for additional pull request review items.
+            Please note that the `CHANGELOG.md` file contents are generated automatically. This is to prevent pull request merge conflicts, especially for contributions which may not be merged immediately. Please see the [Contributing Guide](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/pullrequest-submission-and-lifecycle.md#changelog-process) for additional pull request review items.
```

It looks like the link was updated, but not the description for it:

```suggestion
            Please note that the `CHANGELOG.md` file contents are generated automatically. This is to prevent pull request merge conflicts, especially for contributions which may not be merged immediately. Please see the [Contributing Guide](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/pullrequest-submission-and-lifecycle.md#changelog-process) for additional information on adding CHANGELOG entries.
```